### ISSUE TITLE
Module dependency issue

### ISSUE THREAD

**Issue by kbrady-vibrato** [cloudposse/terraform-aws-cloudfront-s3-cdn#104]
Hi,

Getting the follow error on apply when using 0.34.1:

```
Error: Error putting S3 policy: OperationAborted: A conflicting conditional operation is currently in progress against this resource. Please try again.
	status code: 409, request id: 2BF044972A48C379, host id: JvEoMTj12PM1d1JciiwtkAbR+Nwl+J2ToLiSqCZvKQIMm3uj/KxanCulfFuPbTMG7SqNDKfunpQ=

  on .terraform/modules/thismodule.cdn/main.tf line 99, in resource "aws_s3_bucket_policy" "default":
  99: resource "aws_s3_bucket_policy" "default" {
```

Has anyone else seen this?

Thanks

**Comment by nitrocode** [comment]
What are the module inputs ?

**Comment by Nuru** [comment]
Fixed by #136
